Gifski/MainWindowController.swift

```diff
@@ -186,6 +217,30 @@ final class MainWindowController: NSWindowController {
 			}
 		}
 	}
+
+	private func setupTimeRemainingLabel() {
+		guard let view = view else {
+			return
+		}
+
+		timeRemainingLabel.translatesAutoresizingMaskIntoConstraints = false
+
+		NSLayoutConstraint.activate([
+			timeRemainingLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+			timeRemainingLabel.topAnchor.constraint(equalTo: circularProgress.bottomAnchor)
+		])
+	}
+
+	private func formattedTimeRemaining(seconds: TimeInterval) -> String? {
+		elapsedTimeFormatter.allowedUnits = seconds < 60 ? .second : .minute
```

Comment:
It should still show hours.

```suggestion
		elapsedTimeFormatter.allowedUnits = seconds < 60 ? .second : [.hour, .minute]
```